app/models/claim.rb

```diff
@@ -62,6 +70,14 @@ def any_adjustments?
 
   private
 
+  def granted_and_allowed_less_than_claim
+    # TODO: refactor this to summed_costs.dig(:allowed_gross_cost, :text, :value)
```

Comment:
Nit: I don't like this `TODO`. If refactoring is the right thing to do, then either we should do the refactoring, or if it's too big a piece of work to do in this ticket, we should create another ticket and reference the number of that ticket in this comment, so that it's clear when and how the TODO is to be resolved.